Genestealer Cult/Readme.md

```diff
@@ -0,0 +1,48 @@
+# Genestealer Cult Index v1.0
+## General Information
+* This Index contains all units from the current `Index: Genestealer Cult`
+  
+* **`Leaders`** are all separate unit profiles acting as if they were a sole unit on the battlefield, they include their **`Leader`** ability but this should be left inactive unless simulating that effect.
+  
+* All units that have the `Damaged: 1-X Wounds Remaining` ability have a `-1 to hit roll (if attacker)` ability in their unit profile that can be enabled.
+  
+* All unit profiles have the weapons shown on their datasheet unless specified.
+  
+* Some Units have Abilities in the form of a Weapon Profile these will have `- (Ability)` at the end of their weapon name.\
+
+## Army Rule
+* **`Power From Pain:`** <br> Add the following ability to the Global Modifiers:
+    * `Re-roll all possible failed hit rolls (if attacker)`
+
+    
+## Units - Standard
+
+* #### Abominant:
+  * **`The Chosen One:`** *Leader Ability* <br> Enable the profile ability `SUSTAINED HITS 1`.
+
+* #### Biophagus:
+  * **`Twisted Science:`** *Leader Ability* <br> Enable the profile ability `LETHAL HITS`.
+
+* #### Magus:
+  * **`Bodyguard:`** *Leader Ability* <br> Enable the profile ability `Feel no pain 4+`.
```

Comment:
#### Magus
* Change `Bodyguard` to `Spiritual Leader` also change `Feel no pain 4+` to `5+`